Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM opensuse/leap:15.2
+
+RUN zypper refresh \
+    && zypper install -y \
+        osc \
+        perl \
+        perl-YAML-LibYAML \
+        perl-XML-Simple \
+        perl-Parse-CPAN-Packages \
+        procmail \
+        wget \
+        vim \
+        git \
+        perl-Text-Autoformat \
+        perl-YAML \
+        perl-Pod-POM \
+        perl-libwww-perl \
+        perl-Class-Accessor-Chained \
+        perl-Perl-PrereqScanner \
+        perl-Algorithm-Diff \
+        perl-Module-Build-Tiny \
+        perl-ExtUtils-Depends \
+        perl-ExtUtils-PkgConfig \
+        obs-service-format_spec_file \
+    && true
+
+
+RUN cd /tmp && wget http://www.cpan.org/modules/02packages.details.txt.gz
```

Comment:
Consecutive `RUN`s should be avoided, see https://github.com/hadolint/hadolint/commit/4cb8f28ebecfd0343241369952fc9b3e7e4ea866.

Reply:
I think I will keep it for now, though.
The image is not being uploaded currently, and I might extend the Dockerfile in the future, and I find it annoying when it has to rerun the whole `RUN` because I added another command.